modules/forum/src/main/MentionNotifier.scala

```diff
@@ -49,7 +49,7 @@ final class MentionNotifier(
           .filterExists(candidates take 10)
           .map(_.take(5).toSet)
       mentionableUsers <- prefApi.mentionableIds(existingUsers)
-      users <- LilaFuture.filterNot(mentionableUsers.toList) { relationApi.fetchBlocks(_, mentionedBy) }
+      users            <- mentionableUsers.toList.filterA(relationApi.fetchBlocks(_, mentionedBy))
```

Comment:
missing a negation?

Reply:
oops, should be `mentionableUsers.toList.filterA(!relationApi.fetchBlocks(_, mentionedBy))`, will fix it.